sno/git_util.py

```diff
@@ -6,6 +6,38 @@
 from .timestamps import tz_offset_to_minutes
 
 
+def get_head_tree(repo):
+    """Returns the tree at the current repo HEAD."""
+    if repo.is_empty:
+        return None
+    try:
+        return repo.head.peel(pygit2.Tree)
+    except pygit2.GitError:
+        # This happens when the repo is not empty, but the current HEAD has no commits.
+        return None
+
+
+def get_head_commit(repo):
+    """Returns the commit at the current repo HEAD."""
```

Comment:
Can the docstring be expanded with the comments about when/why `None` is returned?

Reply:
Done